build.gradle

```diff
@@ -7,6 +7,7 @@ testSets {
     testIntegration
     testMessageIntegration
     testMessageIntegrationWithReviewPeriod
+    testDlq
```

Comment:
Separated DLQ test due to Localstack Md5 hash bug of confusing messages (effects custom attributes)

I have not included this test in the pipeline as it will require spinning up local stack to test a library dependency. 
I have included it in the `local_checks` script